ReactiveXComponentTest/WebSocket/WebSocketTests.cs

```diff
@@ -249,5 +249,67 @@ public void SnapshotTest()
                 Check.That(snapshotReceived).IsTrue();
             }
         }
+
+        [Test]

```

Comment:
For tests we need to mock a response and make sure we properly handle it. Have a look at the snapshot test as an example. Because making sure that we properly format the request is good but just not enough.